src/components/activity/ActivityDirectivesTable.svelte

```diff
@@ -210,6 +335,30 @@
       <ContextMenu.Item size="sm" on:click={scrollTimelineToActivityDirective}>Scroll to Activity</ContextMenu.Item>
       <ContextMenu.Separator />
     {/if}
+  </svelte:fragment>
+
+  <svelte:fragment slot="context-menu-alt">
+    {#if showPackLeftMenu}
```

Comment:
Instead of defining a new slot, this can just be moved directly under line 337 of this file.